microdata_tools/validation/components/unit_type_variables/TRAFIKKULYKKE_PERSON.json

```diff
@@ -0,0 +1,61 @@
+{
+  "shortName": "PERSON_TRAFIKKULYKKE_ID",
+  "name": [
+    {
+      "languageCode": "no",
+      "value": "Person i trafikkulykkeidentifikator"
+    },
+    {
+      "languageCode": "en",
+      "value": "Person in road traffic accident identifier"
+    }
+  ],
+  "description": [
+    {
+      "languageCode": "no",
+      "value": "Identifikator for personer involvert i trafikkulykker i microdata"
+    },
+    {
+      "languageCode": "no",
+      "value": "Identifier for persons involved in road traffic accidents in microdata"
+    }
+  ],
+  "dataType": "STRING",
+  "unitType": {
+    "requiresPseudonymization": true,
+    "shortName": "PERS_TRAFIKKULYKKE",
```

Comment:
To follow the convention, I suggest "PERSON_TRAFIKKULYKKE" as the `shortName` here.